helm/examples/local/README.md

```diff
@@ -2,12 +2,12 @@
 
 ## Directory contents
 
-### hpcc-localfile/
+### localfile/
 
-The hpcc-localfile helm chart will provision a new Persistent Volume and a Persistent Volume Claim for each of the required HPCC storage types.
+The hpcc systems localfile helm chart will provision a new Persistent Volume and a Persistent Volume Claim for each of the required HPCC storage types.
```

Comment:
the chart name is called "hpcc-localfile", so should keep that there, e.g. change to:
"The hpcc systems hpcc-localfile helm chart will... "